pytest_tests/helpers/s3_helper.py

```diff
@@ -127,3 +130,27 @@ def assert_object_lock_mode(
         assert (
             retain_date - last_modify + timedelta(seconds=1)
         ).days == retain_period, f"Expected retention period is {retain_period} days"
+
+
+def assert_s3_acl(acl_grants: dict, permitted_users: str):
+    if permitted_users == "AllUsers":
+        for i in range(len(acl_grants)):
+            if acl_grants[i].get("Grantee", {}).get("URI") is not None:
+                uri = acl_grants[i].get("Grantee", {}).get("URI")
+                permission = acl_grants[i].get("Permission")
+        assert (uri, permission) == (
```

Comment:
check only last grant, not all grands in list.

Reply:
I thought it would be right to check if there is no block (left by any chance\bag) granting full control to all users if we expect only canonical user(-s) to have full control.